packs/spell-effects/spell-effect-blood-vendetta-failure.json

```diff
@@ -4,7 +4,7 @@
     "name": "Spell Effect: Blood Vendetta (Failure)",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Blood Vendetta]</p>\n<p>Until the bleeding stops, the target has weakness 1 to piercing and slashing damage.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Blood Vendetta]</p>\n<p>You gain weakness 1 to piercing and slashing damage.</p>"
```

Comment:
Consider adding a predicate for `"self:condition:persistent-damage:bleed"`, and including the "Until the bleeding stops" in the description.